### PR TITLE
finished milestone 0

### PR DESCRIPTION
**Pull request by AutumnFoto**
## Description
-auth working, logo, logout working, webpack and firebase setup, navbar up and working!

## Related Issue
#1 #2 #3 #4 

## Motivation and Context
finished milestone 0

## How Can This Be Tested?

git fetch development

## Screenshots (if appropriate):
deploy link : https://console.firebase.google.com/project/pinterestproject-d87f5/overview  
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/javascripts/helpers/viewHelper.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// import firebase from 'firebase/app';
```
Comment (drteresavasquez) [inline src/javascripts/helpers/viewHelper.js:1]: If you are not using it, remove it from production code please. No commented code.

File: src/javascripts/main.js [hunk 2]
```diff
@@ -1,8 +1,12 @@
+import firebase from 'firebase/app';
+import apiKeys from './helpers/data/apiKeys.json';
+import authData from './helpers/data/authData';
+// import navBar from './components/navBar/navBar';
```
Comment (drteresavasquez) [inline src/javascripts/main.js:4]: remove

**Review by drteresavasquez — CHANGES_REQUESTED** [review]
Remove the commented code, and get me a deploy link and I will take another look.

File: src/javascripts/helpers/viewHelper.js (hunk 1 above)
Comment (AutumnFoto) [reply src/javascripts/helpers/viewHelper.js:1]: will do!

**Pull request merged**